### PR TITLE
Add rootcheck to RBAC reference.

### PR DESCRIPTION
**Pull request by Selutario**
## Description

Hello team! This PR is related to [#6329](https://github.com/wazuh/wazuh/issues/6329). New actions have been added to API RBAC, so the reference page needed to be updated.

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 
- [x] Used impersonal speech. 
- [x] Used uppercase only on nouns. 
- [x] Updated the `redirect.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

Regards,
Selu.

### CODE REVIEW

File: source/user-manual/api/rbac/reference.rst [hunk 1]
```diff
@@ -482,6 +495,21 @@ mitre:read
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 - :api-ref:`GET /mitre <operation/api.controllers.mitre_controller.get_attack>` (`*:*`_)
 
+Rootcheck
```
Comment (davidjiglesias) [inline source/user-manual/api/rbac/reference.rst:498]: Also add agent:group (I know it is not present on the others, but a PR has been included to 4.0.1 to add this, so we need to add it to new additions)

**Review by davidjiglesias — CHANGES_REQUESTED** [review]
We are missing changes to equivalence table. Remember the equivalence table displays the changes from 3.X to 4.X.

Rootcheck endpoints were present in 3.X, so we need to adapt the table after adding them back, some of the old endpoints are still deprecated, but others are not.

File: source/user-manual/api/equivalence.rst [hunk 2]
```diff
@@ -466,28 +466,29 @@ Decoders
...
+| Clear rootcheck database             | ``DELETE /rootcheck``                   | :api-ref:`DELETE /rootcheck <operation/api.controllers.rootcheck_controller.delete_rootcheck>`                      | Added ``agents_list`` parameter in query.                                                                                                                                 |
++--------------------------------------+-----------------------------------------+---------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Clear rootcheck database of an agent | ``DELETE /rootcheck/{agent_id}``        | :api-ref:`DELETE /rootcheck <operation/api.controllers.rootcheck_controller.delete_rootcheck>`                      | Use the ``agents_list`` parameter to indicate which rootcheck databases must be deleted.                                                                                  |
```
Comment (davidjiglesias) [inline source/user-manual/api/equivalence.rst:477]: API 4.1 usage is wrong

File: source/user-manual/api/equivalence.rst [hunk 3]
```diff
@@ -466,28 +466,29 @@ Decoders
...
+| Run rootcheck scan in all agents     | ``PUT /rootcheck``                      | :api-ref:`PUT /rootcheck <operation/api.controllers.rootcheck_controller.put_rootcheck>`                            | Added ``agents_list`` parameter in query.                                                                                                                                 |
++--------------------------------------+-----------------------------------------+---------------------------------------------------------------------------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
+| Run rootcheck scan in an agent       | ``PUT /rootcheck/{agent_id}``           | :api-ref:`PUT /rootcheck <operation/api.controllers.rootcheck_controller.put_rootcheck>`                            | Use the ``agents_list`` parameter to indicate which agents must run a rootcheck scan.                                                                                     |
```
Comment (davidjiglesias) [inline source/user-manual/api/equivalence.rst:490]: API 4.1 usage is wrong

File: source/user-manual/api/rbac/reference.rst [hunk 4]
```diff
@@ -767,6 +795,34 @@ Resources
...
 
+rootcheck_read
+^^^^^^^^^^^^^^^
+Allow read all rootcheck information.
+
+Actions
+    - `rootcheck:read`_
+
+Resources
+    - ``agent:id:*``
```
Comment (davidjiglesias) [inline source/user-manual/api/rbac/reference.rst:806]: agent:group:* ??

File: source/user-manual/api/rbac/reference.rst [hunk 5]
```diff
@@ -767,6 +795,34 @@ Resources
...
+^^^^^^^^^^^^^^^
+Allow read, run and clear rootcheck information.
+
+Actions
+    - `rootcheck:read`_
+    - `rootcheck:clear`_
+    - `rootcheck:run`_
+
+Resources
+    - ``agent:id:*``
```
Comment (davidjiglesias) [inline source/user-manual/api/rbac/reference.rst:821]: agent:group:* ?

**Review by davidjiglesias — CHANGES_REQUESTED** [review]

**Review by davidjiglesias — APPROVED** [review]

**Review by davidfmiranda — APPROVED** [review]
LGTM

**Pull request merged**